sdfObject/sdfthing-oma-device.sdf.json

```diff
@@ -135,6 +135,14 @@
             "description": "Total amount of storage space which can store data and software in the LwM2M Device (expressed in kilobytes).  Note: 1 kilobyte corresponds to 1000 bytes.",
             "writable": false,
             "type": "integer"
+          },
+          "ExtDevInfo": {
+            "label": "ExtDevInfo",
+            "description": "Reference to external \"Device\" object instance containing information. For example, such an external device can be a Host Device, which is a device into which the Device containing the LwM2M client is embedded. This Resource may be used to retrieve information about the Host Device.",
+            "sdfType": "link",
```

Comment:
Looks good! To enable using `properties` quality, just add also quality
`"type" : "object"`
(Yes, we need to update the schema to catch that. Thanks!)

Reply:
Fixed